org.mixedrealitytoolkit.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboardTouchAdapter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Mixed Reality Toolkit Contributors
+// Licensed under the BSD 3-Clauseusing System;
+
+using Microsoft.MixedReality.GraphicsTools;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace MixedReality.Toolkit.UX.Experimental
+{
+    public class NonNativeKeyboardTouchAdapter : MonoBehaviour
+    {
+        private void Awake()
```

Comment:
As a best practive we are requiring that MonoBehaviour event functions be at least "protected", if the class is not sealed.

This requirement is meant to prevent subclasses from accidentally hiding the base's implementation of event functions (Awake, OnEnable, ect.).  That is, if the base event functions are private and the subclass implemented its own version, the compiler will happily allow this (no warning, no error).   This would then hide the base implementation. 

If the base event functions are protected, the compiler will at least provide a warning message.

So please make `Awake` , `OnEnable`, and `Update` projected.

Reply:
Done